MAINTAINERS.md

```diff
@@ -6,28 +6,27 @@ such as architecture, process, existing issues.
+| Dmitriy Creed         | [@Cre-eD](https://github.com/Cre-eD)                 | creed@soramitsu.co.jp           | DevSecOps                                                                     |
+| William Richter       | [@WRRicht3r](https://github.com/WRRicht3r)           | ryan.rrr66@gmail.com            | Documentation                                                                 |
+| Andrey Kostiuchenko   | [@arndey](https://github.com/arndey)                 | kostiuchenko@soramitsu.co.jp    | Developer: Java, Kotlin                                                       |
+| Artemii Gerasimovich  | [@QuentinI](https://github.com/QuentinI)             | gerasimovich@soramitsu.co.jp    | Development: Rust                                                             |
+| Daniil Polyakov       | [@Arjentix](https://github.com/Arjentix)             | daniil.polyakov@soramitsu.co.jp | Development: Rust                                                             |
+| Dmitry Balashov       | [@0x009922](https://github.com/0x009922)             | dbalashov@soramitsu.co.jp       | Development: Rust, TypeScript, JavaScript                                     |
+| Ekaterina Mekhnetsova | [@outoftardis](https://github.com/outoftardis)       | mekhnetsova@soramitsu.co.jp     | Documentation                                                                 |
+| Grzegorz Bazior       | [@baziorek](https://github.com/baziorek)             | bazior@agh.edu.pl               | Development: C++, Python, JavaScript, TypeScript                              |
+| Ilia Churin           | [@ilchu](https://github.com/ilchu)                   | churin@soramitsu.co.jp          | Development: Rust                                                             |
+| Marin Veršić          | [@mversic](https://github.com/mversic)               | versic@soramitsu.co.jp          | Development: Rust                                                             |
+| Sam H. Smith          | [@SamHSmith](https://github.com/SamHSmith)           | smith@soramitsu.co.jp           | Development: Rust, C                                                          |
+| Shanin Roman          | [@Erigara](https://github.com/Erigara)               | shanin@soramitsu.co.jp          | Development: Rust                                                             |
+| Shunkichi Sato        | [@s8sato](https://github.com/s8sato)                 | s.sato@soramitsu.co.jp          | Development: Rust                                                             |
+| Timur Guskov          | [@gv-timur](https://github.com/gv-timur)             | guskov@soramitsu.co.jp          | Development: Java                                                             |
+| Vasily Zyabkin        | [@BAStos525](https://github.com/BAStos525)           | zyabkin@soramitsu.co.jp         | DevOps                                                                        |
```

Comment:
same as for doc maintainers: it makes more sense to list all developers next to each other, and others -- before or after

I'd also group them based on language where possible (at least those who only work on one sdk, e.g. java)